deepchem/data/tests/test_dftdata.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pathlib
+import pytest
+try:
+    from deepchem.data.dft_data import DFTEntry, DFTSystem, load_entries
+except ModuleNotFoundError:
+    raise ModuleNotFoundError("This utility requires dqc")
+import torch
+
+
+@pytest.fixture
```

Comment:
Let's not use pytest fixtures. We generally avoid fixture usage in the unit tests since it means that unit tests aren't self-contained. Instead just explicitly load things in each test